lib/services/ws-service.js

```diff
@@ -151,7 +151,7 @@ function webSocketServiceFactory(
     };
 
     WebSocketService.prototype._forwardResourceMethod = function (method, msg, wsConn, rawMsg) {
-        logger.info('Client: %s requested a resource call.'.format(wsConn.id),
+        logger.debug('Client: %s requested a resource call.'.format(wsConn.id),
```

Comment:
@rolandpoulter this sequence seemed more relevant to `debug` than `info`, so I'm suggesting we switch them over. That work for you?